src/select/styled-components.js

```diff
@@ -31,7 +31,7 @@ export const Input = styled('input', props => {
     sizing: {scale300},
   } = $theme;
   return {
-    ...getInputStyles({...props, $size: SIZE.default}),
+    ...getInputStyles({...props, $size: SIZE.default, $disabled: false}),
```

Comment:
The disabled is false by default. It looks weird that we use disabled input as a select in the first place and this change in addition to it can be confusing. I also added a change based on a valid comment from @schnerd and @abmai to move the cursor style on input out of `:hover`.

Reply:
ok, I am removing this at all here.
However, Select needs Input in disabled state for read-only purpose. It totally complies with behavior of Input, functionally it is correct. Agree that Select is not in disabled state, but Input is not internal part of it, it's just reused by Select, so it can be adjusted to whatever needs by external Select consumer, which is read-only mode in this case. We already discussed amount of efforts and possible caveats to not set Input in disabled state and provide our own implementation instead. And seems like this effort is to be postponed for future improvements.

Reply:
I believe moving out the cursor style out of `:hover` hasn't fixed the issue here, so you still need to do the override for it below.